src/main/java/domain/Car.java

```diff
@@ -0,0 +1,39 @@
+package domain;
+
+public class Car {
+
+    public static final int DEFAULT_POSITION = 0;
+    public static final int MAX_NAME_LENGTH = 5;
```

Comment:
```suggestion
    private static final int DEFAULT_POSITION = 0;
    private static final int MAX_NAME_LENGTH = 5;
```
`Car`에서만 사용되는 상수는 private으로 접근을 막으면 좋겠네요.